chain/pool/txpool.go

```diff
@@ -23,8 +24,9 @@ var (
 	ErrDoubleSpend           = errors.New("IsDoubleSpend check faild")
 	ErrTxnType               = errors.New("invalidate transaction payload type")
 	ErrNonceTooLow           = errors.New("nonce is too low")
-	ErrDuplicateName         = errors.New("Duplicate NameService operation in one block")
-	ErrNoNameRegistered      = errors.New("name already has not be registered")
+	ErrDuplicateName         = errors.New("name already has be registered")
+	ErrDuplicateRegistrant   = errors.New("Registrant already has registered name")
```

Comment:
=> "Registrant has already registered this name"

Reply:
ok